src-tauri/Cargo.toml

```diff
@@ -95,6 +95,7 @@ hex = "0.4.3"
 openssl = { version = "0.10", features = ["vendored"] }
 ring-compat = "0.8.0"
 der = "0.7.9"
+psp = { git = "https://github.com/Misieq01/psp.git", rev = "ddef0c6" }
```

Comment:
If we're running a forked version of the plugin it'll need to be hosted under the `tari-projects` organisation, and we need an explanation of why we're using the forked version.